### DISCUSSION TITLE
Add support for viewing group participants

### DISCUSSION THREAD

**Discussion by Docbroke** [d99kris/nchat#304] [Ideas]
command to view list of participants in group, we may allow sending message from this list, which will solve request in another discussion.

**Comment by d99kris** [comment]
Hi @Docbroke - thanks for suggesting a feature. Would it be sufficient to have a list dialog shown, similar to ctrl-n / alt-n showing the participants and allowing starting/opening chat with them?

**Comment by Docbroke** [comment]
Yes, that will be sufficient.

**Reply by d99kris** [reply]
Cool thanks, will track this under #298 and close this one.
